lib/systems/default.nix

```diff
@@ -100,7 +100,12 @@ rec {
         }.${final.parsed.kernel.name} or null;
 
          # uname -m
-         processor = final.parsed.cpu.name;
+         processor =
+           if final.isPower64
+           then "ppc64${lib.optionalString final.isLittleEndian "le"}"
```

Comment:
https://github.com/NixOS/nixpkgs/blob/1702e9d2d42480d994aa694d9405b33ac6c85abb/lib/systems/default.nix#L134-L143
should the powerpc64{,le} match be removed from there

Reply:
Excellent idea.

Reply:
What about the ppc

Reply:
Thanks for catching that.